### PR TITLE
Add order to Col

### PR DESCRIPTION
**Pull request by Aleksei2507**
Hello! This is my first open-source PR )) I tried to do something useful )) I added the order to the columns, the little test, and edited the docs/types =)

### CODE REVIEW

File: src/components/grid/col.js [hunk 1]
```diff
@@ -34,6 +34,15 @@ const Col = styled.div`
     `)}
   `}
 
+  ${p => p.order && css`
```
Comment (santosfrancisco) [inline src/components/grid/col.js:37]: here, I suggest these changes to work with numerical values ​​including negative numbers
```suggestion
  ${p => p.order && css`
    ${DIMENSIONS.map(d =>
    config(p).breakpoints[d] && config(p).media[d]`
    ${typeof p.order === 'object'
    ? p.order[d] !== undefined && `order: ${p.order[d]}`
    : `order: ${p.order};`};
    `)}
  `}
```

**Review by santosfrancisco — CHANGES_REQUESTED** [review]
@Aleksei2507 , thank you very much for your contribution! I only made a few suggestions to make it easier to use. Any questions, I'll be here. Also, it would be nice to have usage documentation like the `offset` and` alignment` sections

File: src/components/grid/col.js (hunk 1 above)
Comment (Aleksei2507) [reply src/components/grid/col.js:37]: Oh! Yes, you right! I will fix

**Comment by Aleksei2507** [comment]
> @Aleksei2507 , thank you very much for your contribution! I only made a few suggestions to make it easier to use. Any questions, I'll be here. Also, it would be nice to have usage documentation like the `offset` and` alignment` sections

I added an example for order

File: src/components/grid/col.js (hunk 1 above)
Comment (santosfrancisco) [inline src/components/grid/col.js:37]:
```suggestion
  ${p => p.order !== undefined && css`
```

**Review by santosfrancisco — CHANGES_REQUESTED** [review]
@Aleksei2507 , I'm sorry for the delay, but it was pretty busy here where I work. Finally I was able to test calmly and left just one more change suggestion. When I tried to add `order = {0}` I had a small problem with css. So I suggested doing the order verification a little differently. Thank you for your patience and contribution

File: src/components/grid/col.js (hunk 1 above)
Comment (Aleksei2507) [reply src/components/grid/col.js]: Don't worry )) Work is work! ))
Wow... That's a good point ))

I fixed =)

**Review by santosfrancisco — APPROVED** [review]

**Pull request merged**